src/EmbeddedVsix/Manifests/PreVS2022/source.extension.vsixmanifest

```diff
@@ -15,6 +15,6 @@
         <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%|" />
     </Assets>
     <Prerequisites>
-        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[14.0,18.0)" DisplayName="Visual Studio core editor" />
+        <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[16.0,17.0)" DisplayName="Visual Studio core editor" />
```

Comment:
Strictly speaking it isn't necessary to update the supported versions for this prerequisite. However, this manifest is only used for VS2019 now, so we might as well make it match the expected range for VS2019.

Reply:
Thank you for fixing it!